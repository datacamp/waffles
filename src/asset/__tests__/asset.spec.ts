import * as Assets from '../index';

describe('Asset', () => {
  it('exports every asset', () => {
    expect(Assets).toMatchSnapshot();
  });
});
