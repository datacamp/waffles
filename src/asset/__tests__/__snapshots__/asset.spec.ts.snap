// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Asset exports every asset 1`] = `
Object {
  "AvatarPlaceholderColored": [Function],
  "BarChart3D": [Function],
  "Bookmark3D": [Function],
  "Branch3D": [Function],
  "Calendar3D": [Function],
  "Certification3D": [Function],
  "Checkmark3D": [Function],
  "Code3D": [Function],
  "Database3D": [Function],
  "Direction3D": [Function],
  "DoubleArrow3D": [Function],
  "Edit3D": [Function],
  "Education3D": [Function],
  "Email3D": [Function],
  "Finance3D": [Function],
  "FourArrows3D": [Function],
  "Globe3D": [Function],
  "Import3D": [Function],
  "Laptop3D": [Function],
  "Love3D": [Function],
  "MagnifyingGlass3D": [Function],
  "MailboxEmpty3D": [Function],
  "Mobile3D": [Function],
  "Paper3D": [Function],
  "Path3D": [Function],
  "People3D": [Function],
  "Rocket3D": [Function],
  "Share3D": [Function],
  "Target3D": [Function],
  "TargetArrow3D": [Function],
  "Team3D": [Function],
  "Timer3D": [Function],
  "TimerSleep3D": [Function],
  "Track3D": [Function],
  "Trophy3D": [Function],
}
`;
